llvm/lib/CodeGen/MachineBasicBlock.cpp

```diff
@@ -426,10 +427,8 @@ void MachineBasicBlock::print(raw_ostream &OS, ModuleSlotTracker &MST,
     OS.indent(2) << "liveins: ";
 
     ListSeparator LS;
-    for (const auto &LI : liveins()) {
-      OS << LS << printReg(LI.PhysReg, TRI);
-      if (!LI.LaneMask.all())
-        OS << ":0x" << PrintLaneMask(LI.LaneMask);
+    for (const MCRegister Reg : liveins()) {
```

Comment:
Remove the `const`. (Pointers to const and const references are good, but plain const is not.)